### ISSUE TITLE
Timestamps constants on Trend Metrics

### ISSUE THREAD

**Issue by rodrigoo-csantos** [laravel/nova-issues#1273]
- Laravel Version: 5.7.22
- Nova Version: 1.3.1
- PHP Version: 7.2
- Operating System and Version: W10

### Description:

Trend Metrics are not considering CREATED_AT constants recorded in models

Value Metrics are Ok !

### Steps To Reproduce:

const on Model
![image](https://user-images.githubusercontent.com/22321434/51676535-046cd780-1fbe-11e9-8a77-7c3e1e378169.png)

Error on Metric request
![image](https://user-images.githubusercontent.com/22321434/51676570-1b132e80-1fbe-11e9-816d-f695a88137ac.png)

> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'created_at' in 'field list' (SQL: select date_format(`created_at` - INTERVAL 2 HOUR, '%Y-%m-%d') as date_result, count(`propostas`.`id`) as aggregate from `propostas` where `created_at` between 2019-01-18 00:00:00 and 2019-01-24 09:54:42 group by date_format(`created_at` - INTERVAL 2 HOUR, '%Y-%m-%d') order by `date_result` asc)

My Dashboard
![image](https://user-images.githubusercontent.com/22321434/51676612-4b5acd00-1fbe-11e9-9f4d-590f01b919da.png)

My Metric class
![image](https://user-images.githubusercontent.com/22321434/51676627-59a8e900-1fbe-11e9-8026-42238a165a30.png)

**Comment by dkulyk** [comment]
You need to specify it in `countByDays($request, $model, $column = 'created_at')`

**Comment by dkulyk** [comment]
PR was sent to upstream.

**Comment by blueyama** [comment]
Yep noticed the same thing. I also specified the `CREATED_AT` constant in my model, and as mentioned above its being ignored. Currently it's a hardcoded string `'created_at'`. The code for all metrics appears like so:

```
    public function countByWeeks($request, $model, $column = 'created_at')
    {
        return $this->count($request, $model, self::BY_WEEKS, $column);
    }
```

A quick fix that appears to work:
```
    public function countByWeeks($request, $model, $column = null)
    {
        return $this->count($request, $model, self::BY_WEEKS, $column ?:$model::CREATED_AT);
    }
```

In the meantime I'll pass the value as the 3rd param but looking forward to this bug fix.

**Comment by dkulyk** [comment]
This has been fixed and will be included in the next release.

**Comment by usamamuneerchaudhary** [comment]
Hasn't been fixed in Nova 2.6.1
